### PR TITLE
Issue/20 countdown game viewcontroller

### PR DESCRIPTION
**Pull request by wassupnari**
This PR implements the CountdownGameViewController. It also contains game_room `id` fix, and I updated game_room `name` field as selected game category, so it looks like below:

screen shot 2016-11-27 at 4 35 50 pm

There's a bug where selected picker doesn't return the value correctly, and I couldn't figure out why. For example, in GameOptionsViewController, when I selected "YAY CLASS" as my game category, but `categoryPickerData[row]` in delegate returns something else. It doesn't happen all the time, but pretty often. Let me know if you guys have any idea why this is happening.

### CODE REVIEW

File: ios-trivia-game/ViewControllers/GameOptionsViewController.swift [hunk 1]
```diff
@@ -77,15 +78,20 @@ class GameOptionsViewController: UIViewController {
...
     // Make an api call to create a game room, and then go to CountdownTimerViewController if it succeeds.
     @IBAction func onStartGameClicked(_ sender: Any) {
-        print("game started clicked")
         var newGame = ["max_num_of_questions" : numOfQuestions ?? 0,
+                       "id" : FirebaseClient.instance.createGameRoomId().key,
                    "current_question" : 0,
                    "is_public" : isPublic ?? true,
                    "max_num_people" : numOfPlayers ?? 5,
-                   "name" : "test",
+                   "name" : self.selectedCategory ?? "",
```
Comment (saviotsui) [inline ios-trivia-game/ViewControllers/GameOptionsViewController.swift]: I was expecting the name to be something the user inputs that let's other players know what game it is.  Like a chatroom name.

**Review by saviotsui — CHANGES_REQUESTED** [review]
Name should be freeform text user input.
Like the name of a chatroom.

File: ios-trivia-game/ViewControllers/GameOptionsViewController.swift (hunk 1 above)
Comment (wassupnari) [reply ios-trivia-game/ViewControllers/GameOptionsViewController.swift]: Okay, I can make that change. I'm thinking adding it in CreateGameViewController, do you think that would work?

Comment (saviotsui) [reply ios-trivia-game/ViewControllers/GameOptionsViewController.swift]: sounds good

**Review by saviotsui — APPROVED** [review]

**Pull request merged**